runtime/commonMain/src/kotlinx/serialization/internal/SerialClassDescImpl.kt

```diff
@@ -101,12 +102,14 @@ public open class SerialClassDescImpl(
         if (this === other) return true
         if (other !is SerialClassDescImpl) return false
         if (serialName != other.serialName) return false
+        // TODO get rid of it during SerialClassDescImpl rework
```

Comment:
are you talking about `elementDescriptors` function?

Reply:
I am talking about comparing each individual element because we already decided that `serialName` is **unique** identifier of SD

Reply:
you can mention that then